src/20_Components/amp-bind.html

```diff
@@ -44,10 +50,13 @@
     <script type="application/json">
       {
         "dog": {
-          "url": "/img/Border_Collie.jpg"
+          "imageUrl": "/img/Border_Collie.jpg",
+          "videoUrl": "/video/dog-video.mp4"
         },
         "cat": {
-          "url": "/img/cat-looking-up-300x200.jpg"
+          "imageUrl": "/img/cat-looking-up-300x200.jpg",
+          "videoUrl": "/video/cat-video.mp4",
+          "styleClass": "redBackground"
```

Comment:
For consistency, can we have a different style for the dogs and set it as the initial background. This will be removed when the class is changed - right? If yes, we should mention this behavior. 

s/styleClass/style